azure-pipelines/signing.yml

```diff
@@ -473,6 +473,36 @@ jobs:
         PathtoPublish: '$(Build.ArtifactStagingDirectory)\drop'
         ArtifactName: 'Drop'
         publishLocation: 'Container'
+    - task: CopyFiles@2
+      displayName: 'Copy Packaging Solution to Staging Area'
+      inputs:
+        SourceFolder: '$(Build.SourcesDirectory)/azure-pipelines/vs-insertion'
+        TargetFolder: '$(Build.ArtifactStagingDirectory)/vs-insertion/staging'
+        Contents: 'vcpkg.nuspec'
+        CleanTargetFolder: true
+    - task: CopyFiles@2

```

Comment:
Why not just start from the drop directly given that the .nuspec is doing the filtering you need?